modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/DynamicInlineScroll.es.js

```diff
@@ -70,8 +70,7 @@ class DynamicInlineScroll extends PortletBase {
 	addListItem_(listElement, pageIndex) {
 		const listItem = document.createElement('li');
 
-		dom.append(
-			listItem,
+		listItem.append(
```

Comment:
I don't think these replacements are correct:

> The ParentNode.append() method inserts a set of Node objects or DOMString objects after the last child of the ParentNode. DOMString objects are inserted as equivalent Text nodes.

`DOMString` objects are inserted as such and not transformed to `Node` objects. Probably need to build the fragments and maybe we can always use `appendChild` so there's no confusion on what the API does and takes.

Maybe this is the reason for the test failures we're seeing consistently in this PR.

/cc @bryceosterhaus, @diegonvs

Reply:
@jbalsas good catch. I'll go through and update.